deployment/gcp/manifest/standalone_cpx.yaml

```diff
@@ -16,11 +16,14 @@ rules:
     resources: ["ingresses", "ingresses/status"]
     verbs: ["*"]
   - apiGroups: ["citrix.com"]
-    resources: ["rewritepolicies"]
+    resources: ["canarycrds", "rewritepolicies"]
     verbs: ["*"]
```

Comment:
Wished to request for making separate section. Please check if it makes sense. I am requesting since different resources may have different verbs need. We are working on removing (*) and it will help in this change as well.
Also, please mention specific verbs which is used by canarycrds

Please perform it at all relevant places.